### PR TITLE
add assignment with '='

### PR DESCRIPTION
**Pull request by dselivanov**
Was surprised to find that `=` assignment does not work :-). There are different R coding styles, some of us prefer `=` over `<-`.

### CODE REVIEW

**Comment by dselivanov** [comment]
Seems tests fail for a reason not related to the PR

**Comment by dirkschumacher** [comment]
Thanks! I wonder if that impacts the `=` assignment operator in named arguments. But since the tests are green, I am happy to merge. 

The other test is from the optimization code. Sometimes that fails to minimize the function properly. I need to fix it (I thought I already had).

File: tests/testthat/test-compile.R [hunk 1]
```diff
@@ -527,3 +527,13 @@ test_that("for loops without a loop variable is deterministic", {
   code2 <- translate(fun, "wat")$cpp_code
   expect_equal(code1, code2)
 })
+
```
Comment (dirkschumacher) [inline tests/testthat/test-compile.R:530]: Would be great if you could remove that extra line here.

**Review by dirkschumacher — CHANGES_REQUESTED** [review]

**Review by dirkschumacher — APPROVED** [review]

**Comment by dirkschumacher** [comment]
Thanks!

**Pull request merged**